apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -54,7 +54,9 @@ const DebugConsole = connect(
     removeWatchExpression,
     evalInCurrentScope: actions.evalInCurrentScope,
     appendLog: actions.appendLog,
-  }
+  },
+  null,
+  {withRef: true}
```

Comment:
It is _very subtly_ stated in the docs.

> In addition to the options passable to connectAdvanced() (see those below), connect() accepts these additional options:

the call to `getWrappedInstace` is in `JsDebugger.jsx`: https://github.com/code-dot-org/code-dot-org/pull/13307/files#diff-ae85fa757658ea370cdd4af9bd3dcb71R334 (that file is collapsed by github by default for some reason, so if you just searched the page for that text before expanding that file, you would have come up empty)